content/account/organizations.textile

```diff
@@ -0,0 +1,193 @@
+** Click on *Billing*.
+** Choose the required package for the selected account and *Save billing details*.
+
+<aside data-type='note'>
+<p>Each account within an Ably Organization is assigned a unique domain. This ensures that environments such as production, staging, and development are properly isolated from one another. </p>
+</aside>
+
+h2. Manage user account access
+
+Manage user account access:
+
+* Open the *Account* navigation dropdown.
+** Click *Organization Users*.
+** Select the user's account access you want to manage.
+** Specify the user's required *Permissions* for the organization.
+** Assign the necessary *Roles* for required *Account* access.
+** Click *Back to Organization users to save*.
+** Add an account name and *Create account*.
+
+<aside data-type='note'>
+<p>Users provisioned through SCIM cannot modify their name or email address within Ably. All personal information updates must be made through your identity provider, and changes will sync to Ably upon the next SCIM update cycle.</p>
+</aside>
+
+h3. User permissions
+
+A user's total permissions consist of:
+
+* Roles and permissions assigned directly to the user.
+* Roles and permissions granted to all groups the user is part of.
+
+h3. Group permissions with SCIM
+
+In the "Okta":#okta setup phase of the SSO and SCIM configuration, you assign users and groups to the new Okta app. The group you created in Okta (your identity provider) is available to view and for user access management within Ably:
+
+* Open the *Account* navigation dropdown.
+** Click *Organization Users*.
+** Under *Ably Realtime identity provider groups*, click *Manage account access*.
+** Select the user's account access you want to manage.
+** Specify the user's required *Permissions* for the organization.
+** Assign the necessary *Roles* for required *Account* access.
+
+
+<aside data-type='note'>
+<p>When you adjust a user's individual roles and permissions, permissions assigned via groups appear greyed out, indicating they cannot be modified individually. </p>
+</aside>
+
+h2. Provision users
+
+Grant new users access within an organization using:
+
+1. "SCIM":#scim.
+2. "Manual":#manual user invitation.
+
+<aside data-type='note'>
+<p>Provisioning via SCIM is the recommended approach. </p>
+</aside>
+
+h2(#scim). Provision users via SCIM
+
+Manage access to multiple Ably accounts through a single identity provider. To enable this, configure both SSO and SCIM. Once configured, Ably automatically provisions and deprovisions users with access to the Ably app in your identity provider, either individually or through assigned groups. New users are added to Ably's default provisioning account with the role of *Developer*.
+
+<aside data-type='warning'>
+<p>Ably will reject any SCIM-provisioned user whose email address hasn't been pre-approved and added to your organization by Ably support. Ensure all users you intend to provision are registered with Ably beforehand.</p>
+</aside>
```

Comment:
```suggestion
<aside data-type='important'>
<p>Ably will reject any SCIM-provisioned user whose email address hasn't been pre-approved and added to your organization by Ably support. Ensure all users you intend to provision are registered with Ably beforehand.</p>
</aside>
```

"by Ably support" - does this not suggest that support have to manually add all users?